Mathlib/Topology/ContinuousFunction/FunctionalCalculus.lean

```diff
@@ -178,10 +178,11 @@ preserved would be burdensome or impossible.
 class UniqueContinuousFunctionalCalculus (R A : Type*) [CommSemiring R] [StarRing R]
     [MetricSpace R] [TopologicalSemiring R] [ContinuousStar R] [Ring A] [StarRing A]
```

Comment:
Can you adjust the docstring of this class, which currently says that it "will have" instances in three cases? It would be nice to say in which file they are as well.